runtime/fdlimits/fdlimits_test.go

```diff
@@ -0,0 +1,31 @@
+package fdlimits_test
+
+import (
+	"syscall"
+	"testing"
+
+	"github.com/prysmaticlabs/prysm/runtime/fdlimits"
+	"github.com/prysmaticlabs/prysm/testing/assert"
+)
+
+func TestSetMaxFdLimits(t *testing.T) {
+	var limit syscall.Rlimit
+	assert.NoError(t, syscall.Getrlimit(syscall.RLIMIT_NOFILE, &limit))
```

Comment:
Will this break tests on Windows?

Reply:
This wouldnt work on windows admittedly. I can add in a check to only run this on linux/macos.

Reply:
Maybe the test could just be to call the method to Raise to the Max, then request Current and compare it to Max? That way the tests stick to the go-ethereum methods (assuming we implicitly trust that these methods are doing the correct thing on each platform).